car/Obstical_Avoidence/Obstical_Avoidence.ino

```diff
@@ -0,0 +1,34 @@
+
+#include <Smartcar.h>
+
+int trigPin = 19; //D19
+int echoPin = 5; //D5
+int MAX_DISTANCE = 300;
+
+SR04 front(trigPin, echoPin, MAX_DISTANCE);
+BrushedMotor leftMotor(smartcarlib::pins::v2::leftMotorPins);
+BrushedMotor rightMotor(smartcarlib::pins::v2::rightMotorPins);
+DifferentialControl control(leftMotor, rightMotor);
+
+SimpleCar car(control);
+
+void setup()
+{
+
+  Serial.begin(9600);
+  
+}
+
+void loop() {
+  
+  int distance = front.getDistance();
+
+  if (distance <= 15){ //stop when distance is less than 15 cm.
```

Comment:
If there is no obstacle, then distance will be `0`. What should your car do then?